sdk/keyvault/azure-keyvault-secrets/src/main/java/com/azure/security/keyvault/secrets/SecretService.java

```diff
@@ -39,7 +39,7 @@
  * This is package-private so that these REST calls are transparent to the user.
  */
 @Host("{url}")
-@ServiceInterface(name = "KeyVaultSecrets")
+@ServiceInterface(name = "KVSecrets")
```

Comment:
I don't like this change. You should change the rule to make it zero to 20 characters, and revert this change.